src/add&delete.js

```diff
@@ -30,8 +31,8 @@ const printTask = (task) => {
 const deleteTask = (li, tasks) => {
   li.children[3].addEventListener('click', () => {
     for (let i = 0; i < tasks.length; i += 1) {
-      if ((tasks[i].index) === (li.children[2].id)) {
-        tasks.splice(i, 1);
+      if ((tasks[i].index) === Number(li.children[2].id)) {
+        tasks.filter((task) => task.completed === true);
```

Comment:
- [ ] Hello I think you made a mistake concerning what the last reviewer was saying. they were stating that you need to use the `filter` method for the clearCompleted function. Here you've used it in the `deleteTask` method. Please remove the filter from this line and replace it with what it had before.